packages/vscode-extension/package.json

```diff
@@ -829,6 +829,71 @@
         "scopeName": "teamsfx-toolkit.output",
         "path": "./syntaxes/teamsfx-toolkit-output.tmLanguage"
       }
+    ],
+    "walkthroughs": [
+      {
+        "id": "teamsToolkitQuickStart",
+        "title": "Get started with Teams toolkit  ",
+        "description": "Jumpstart your Teams app development experience   ",
+        "steps": [
+          {
+            "id": "teamsToolkitCapailities",
+            "title": "What are Teams app \"Capabilities\"",
+            "description": "[Capabilities](https://aka.ms/teamsfx-capabilities-overview) are the extension points for building apps on the Microsoft Teams platform.\n[Watch video (<1 min)](https://aka.ms/teamsfx-video)",
+            "media": {
+              "svg": "media/learn.svg",
+              "altText": "Watch"
```

Comment:
I think better more context for altText, the disabled people may get confused on the content of the video. See https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.02.01#Zguide for more details.